cmd/tkn/main.go

```diff
@@ -26,7 +29,25 @@ func main() {
 	tp := &cli.TektonParams{}
 	tkn := cmd.Root(tp)
 
-	if err := tkn.Execute(); err != nil {
+	args := os.Args[1:]
+	cmd, _, _ := tkn.Find(args)
+	if cmd != nil && cmd == tkn {
```

Comment:
we should write a  test for this shouldn't we?  ' at least a shell test ?

Reply:
@sthaha yeah it's on the desc of the PR :grin: